pkg/agent/handler/handler.go

```diff
@@ -27,6 +27,11 @@ func isStatusFound(rs *v1.ResourceStatus) bool {
 	return true
 }
 
+func isNotStatusSubResourceUpdate(action proto.Event_Type, meta *proto.EventMeta) bool {
+	// TODO update this to check for change in status on resource meta
+	return (action == proto.Event_SUBRESOURCEUPDATED)
```

Comment:
We probably need that meta.SubResource != "status" check here. I know we talked about "state" subresource, once that is added to the resources we need to add that check as well

Reply:
I will add that back